README.md

```diff
@@ -1,37 +1,216 @@
+
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
+|      build_type      | Le type de compilation utilisé. `dev` est la dernière version instable. `stable` est la dernière version stable. `stable` ou `dev` |     `stable` ou `dev`      |
+|        series        | Le type de build utilisé. `dev` est le dernier build unstable : `version.buildType`.                                               | `4.5.stable`, `4.5.1.dev3` |
+
+## Composants
+
+### Alerter
+
+Créer des alertes pour les différentes soumissions dans le système.
+
+```bash
+docker run --name alerter cccs/assemblyline-core python -m assemblyline_core.alerter.run_alerter
+```
+
+### Archiver
+
+Archivage des soumissions, de leurs résultats et des fichiers dans l'archive.
+
+```bash
+docker run --name archiver cccs/assemblyline-core python -m assemblyline_core.archiver.run_archiver
+```
+
+### Dispatcher
+
+Achemine les fichiers dans le système pendant qu'une soumission est en cours. Assurez-vous que tous les fichiers pendant une soumission sont complétés par tous les services requis.
+
+```bash
+docker run --name dispatcher cccs/assemblyline-core python -m assemblyline_core.dispatching
+```
+
+### Expiration
+
+Supprimer les soumissions et leurs résultats lorsque leur TTL expire.
+
+```bash
+docker run --name expiry cccs/assemblyline-core python -m assemblyline_core.expiry.run_expiry
+```
+
+### Ingester
+
+Déplace les fichiers ingérés des files d'attente prioritaires vers les files d'attente de traitement.
+
+```bash
+docker run --name ingester cccs/assemblyline-core python -m assemblyline_core.ingester
+```
+
+### Métriques
+
+Génère des métriques des différents composants du système.
+
+#### Heartbeat Manager
+
+```bash
+docker run --name heartbeat cccs/assemblyline-core python -m assemblyline_core.metrics.run_heartbeat_manager
+```
+
+#### Agrégateur de métriques
+
+```bash
+docker run --name metrics cccs/assemblyline-core python -m assemblyline_core.metrics.run_metrics_aggregator
+```
+
+##### Agrégateur de statistiques
+
+```bash
+docker run --name statistics cccs/assemblyline-core python -m assemblyline_core.metrics.run_statistics_aggregator
+```
+
+### Scaler
+
+Augmente et diminue les services dans le système en fonction de la charge.
+
+```bash
+docker run --name scaler cccs/assemblyline-core python -m assemblyline_core.scaler.run_scaler
+```
+
+### Mise à jour
+
+Assurez-vous que les différents services reçoivent leurs derniers fichiers de mise à jour.
```

Comment:
First of all the tense used for all the components should be consistent for visual. So it should be "Assure" instead of "assurez-vous".